js/downloads.js

```diff
@@ -99,6 +99,14 @@ function initReleaseNotes() {
   }
 }
 
+function onPackageSelect() {
+  var packageSelect = document.getElementById("sparkPackageSelect");
```

Comment:
Oh I missed this; I opened https://github.com/apache/spark-website/pull/76
Only the last line in this method is needed but it won't matter